dataset/dictionary/characters-natlan.ts

```diff
@@ -2552,4 +2561,532 @@ export default [
+    zhCN: "拉威亚",
+    zhTW: "拉威亞",
+    notesEn: "\"Shoal's Haven\" Employee",
+    notes: "のびのびマーケットの「魚たちの家」スタッフ",
+    notesZh: "「鱼群之家」员工",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Macuahuitl",
+    ja: "マゴホイトル",
+    zhCN: "马夸胡伊特尔",
+    zhTW: "馬夸胡伊特爾",   // Note: "馬「夸」胡伊特爾" is the official translation in the game v5.8. Isn't it normal to use "誇"?
+    notesEn: "Owner of \"Asha's Treasure Trove\"",
+    notes: "のびのびマーケットの「アシャの宝庫」店主",
+    notesZh: "「阿夏的宝库」店主",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Nataba",
+    ja: "ナタバ",
+    zhCN: "娜塔芭",
+    zhTW: "娜塔芭",
+    notesEn: "\"Sun\" Seller",
+    notes: "のびのびマーケットの「太陽」の販売員",
+    notesZh: "「太阳」贩卖员",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Motolinia",
+    ja: "ムトリナ",
+    zhCN: "穆托莉娜",
+    zhTW: "穆托莉娜",
+    notesEn: "Owner of \"Floral Whispers\"",
+    notes: "のびのびマーケットの「フローラルウィスパー」店主",
+    notesZh: "「繁花之语」店主",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Tzompantli",
+    ja: "ソパントリ",
+    zhCN: "松潘特利",
+    zhTW: "松潘特利",
+    notesEn: "Owner of \"Street Fighting Fontainian\"",
+    notes: "のびのびマーケットの「ストリート ファイティング フォンテニアン」店主",
+    notesZh: "「灰河街斗士」店主",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Fournier",
+    ja: "ファニエ",
+    zhCN: "富尼耶",
+    zhTW: "富尼耶",
+    notesEn: "Dessert Vendor",
+    notes: "のびのびマーケットのお菓子売り場の店主",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Tlapalli",
+    ja: "ツラッパリ",
+    zhCN: "查帕莉",
+    zhTW: "查帕莉",
+    notesEn: "Owner of \"Me & Dew\"",
+    notes: "ティティ島の「ドランクハニー」店主",
+    notesZh: "「醉蜜」店主",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Tecuhtli",
+    ja: "テクトリ",
+    zhCN: "秦克胡",
+    zhTW: "秦克胡",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Rhythmos",
+    ja: "「リズミカル職人」",
+    zhCN: "「律动魔匠」",
+    zhTW: "「律動魔匠」",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Tobahaq",
+    ja: "トワハック",
+    zhCN: "图瓦哈克",
+    zhTW: "圖瓦哈克",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Nahuatlalli",
+    ja: "ナワトラリ",
+    zhCN: "纳瓦特拉利",
+    zhTW: "納瓦特拉利",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Kahinda",
+    ja: "カシンテ",
+    zhCN: "卡欣妲",
+    zhTW: "卡欣妲",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Mwindo",
+    ja: "ムウェンダ",
+    zhCN: "穆温达",
+    zhTW: "穆溫達",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Ipacura",
+    ja: "イパクラ",
+    zhCN: "伊帕库拉",
+    zhTW: "伊帕庫拉",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Elmi",
+    ja: "エルミ",
+    zhCN: "厄尔蜜",
+    zhTW: "厄爾蜜",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Aroni",
+    ja: "アロニ",
+    zhCN: "阿洛妮",
+    zhTW: "阿洛妮",
+    notes: "ベネットの母親およびアシャ。「燧石の鋒」冒険団団員",
+    tags: [ "natlan", "character-sub" ],
+  },
+  {
+    en: "Macani",
+    ja: "マカンニ",
+    zhCN: "马坎尼",
+    zhTW: "馬坎尼",
+    notes: "ベネットの父親およびアシャ。「燧石の鋒」冒険団団長",
```

Comment:
It sounds a bit strange Japanese. How about removing "およびアシャ" (it also avoids a spoiler) or rewriting to "現在はアシャとなったベネットの父親。"?

```suggestion
    notes: "ベネットの父親。「燧石の鋒」冒険団団長",
```